app/jobs/migrate/populate_fees_with_organization_id_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Migrate
+  class PopulateFeesWithOrganizationIdJob < ApplicationJob
+    queue_as :low_priority
+
+    def perform
+      Fee.where(organization_id: nil).find_in_batches(batch_size: 1000) do |batch|
+        PopulateBatchFeesWithOrganizationIdJob.perform_later(batch.pluck(:id))
```

Comment:
could this be an issue on Redis side if we have to queue millions of uuids with memory and JSON serialization perfomance?

wouldn't be safer to run the migration as follows (one single migration job, no uuid queueing)?

```
        BATCH_SIZE = 1_000
        
        class Fee < ApplicationRecord
          belongs_to :invoice, optional: true
        end

	def perform
          batch = Fee.unscoped.where(organization_id: nil).joins(:invoice).limit(BATCH_SIZE)
	  
	  if batch.exists?
            # rubocop:disable Rails/SkipsModelValidations
	    batch.update_all("organization_id = invoices.organization_id, billing_entity_id = invoices.organization_id")

            # Queue the next batch
            self.class.perform_later
          end
	end
```

Reply:
ah, that's amazing idea!
Thank you!